src/components/LogIn.vue

```diff
@@ -54,13 +55,14 @@ export default defineComponent({
                 url: 'http://localhost:3000/users/login',
                 headers: {
                     'Content-Type': 'application/json',
-                    'Authorization': 'Bearer ' + localStorage.getItem("auth_Token")
+                    'Authorization': 'Bearer ' + store.getters.getToken//localStorage.getItem("auth_Token")
```

Comment:
> Why is the code commented in this part?
I left it commented so everyone could see the change since it could impact other people trying to use the token. I planned to remove the comments in the the next PR



> Would it be better to put the interaction with the back in a common file and just call it by its different parts? For example, if the URL changes, you'll have to change it in a lot of places; if this were common, you wouldn't.

I agree it would be better but right now a lot of api requests are in their own components. I would not spend time changing that right now. Once all functionality is in we can do a PR putting them all in one place